crates/fluvio-spu/src/services/public/stream_fetch_test.rs

```diff
@@ -1274,17 +1274,9 @@ async fn test_stream_aggregate_fetch_multiple_batch(
         .expect("replica");
     ctx.leaders_state().insert(test_id, replica.clone());
 
-    // Aggregate 5 records
-    // These records look like:
-    //
-    // 1
-    // 2
-    // 3
-    // 4
-    // 5
```

Comment:
could you adjust the comments instead of removing them?

Reply:
Yup

Reply:
updated